index.html

```diff
@@ -2,786 +2,838 @@
-                        as Bicep, for Background Imaging of Cosmic Extragalactic<br>
-                        Polarization, reported their results in a scientific briefing at the<br>
-                        Center for Astrophysics here on Monday and in a set of papers<br>
-                        submitted to The Astrophysical Journal.
-                    </p>
+        <div class="second_article">
 
-                </div>
+            <h1 class="second_article_heading">‘As Big as It Gets’</h1>
 
-            </article>
+            <div class="article_paras">
+                <p>
+                    In our own universe, it would serve as a window into the forces<br />
+                    operating at energies forever beyond the reach of particle<br />
+                    accelerators on Earth and yield new insights into gravity itself.
+                    Dr.<br />
+                    Kovac’s ripples would be the first direct observation of<br />
+                    gravitational waves, which, according to Einstein’s theory<br />
+                    of general relativity, should ruffle space-time.
+                    <br /><br />
+                    Marc Kamionkowski of Johns Hopkins University, an early-<br />
+                    universe expert who was not part of the team, said, “This is huge,<br />
+                    as big as it gets.”
+                    <br /><br />
+                    He continued, “This is a signal from the very earliest universe,<br />
+                    sending a telegram encoded in gravitational waves.”
+                    <br /><br />
+                    The ripples manifested themselves as faint spiral patterns in a
+                    bath<br />
+                    of microwave radiation that permeates space and preserves<br />
+                    a picture of the universe when it was 380,000 years old and as hot
+                    as<br />
+                    the surface of the sun.
+                    <br /><br />
+                    Dr. Kovac and his collaborators, working in an experiment known<br />
+                    as Bicep, for Background Imaging of Cosmic Extragalactic<br />
+                    Polarization, reported their results in a scientific briefing at
+                    the<br />
+                    Center for Astrophysics here on Monday and in a set of papers<br />
+                    submitted to The Astrophysical Journal.
+                </p>
+            </div>
 
+        </div>
+        <br>
+        <section>
+            <h2>The Theory of Inflation</h2>
+            <p class="summary-text">
+                <span class="summary-text">Astronomers have found evidence to support the theory of inflation,
+                    which explains how the universe expanded so uniformly and so quickly
+                    in the instant after the Big Bang 13.8 billion years ago.</span>
+            </p>
             <section class="graphics">
```

Comment:
to having 

`

            <section class="graphics">
              <h2>The Theory of Inflation</h2>
              <p class="summary-text">
                  <span class="summary-text">Astronomers have found evidence to support the theory of inflation,
                      which explains how the universe expanded so uniformly and so quickly
                      in the instant after the Big Bang 13.8 billion years ago.</span>
              </p>
<h2 hidden></h2>
                <div class="img-tool">
                    <img src="image.png/img1.png" alt="first image" />
                    <p class="img-p">
                        <strong>THE UNIVERSE</strong> is just under 14 billion years old.
                        From our position in the Milky Way galaxy, we can observe a sphere
                        that is now about 92 billion light-years across. But there's a
                        mystery. Wherever we look, the universe has an even temperature.
                    </p>
                </div>
.....
`
will fix the theory of inflation UI flaw.